tensorrt_llm/_torch/pyexecutor/py_executor_creator.py

```diff
@@ -131,7 +131,16 @@ def create_py_executor(executor_config: ExecutorConfig,
             logger.info(
                 f"Change tokens_per_block to: {executor_config.tokens_per_block} for using FlashMLA"
             )
-        executor_config.kv_cache_config.enable_block_reuse = False
+        if executor_config.kv_cache_config.enable_block_reuse:
+            if is_hopper() or is_blackwell():
+                logger.info(
+                    f"Enable block reuse for MLA, performance may have regression in some cases"
```

Comment:
Shall we add this into the DeepSeek example doc instead of the log msg here? 
I don't think one will look into this info level msg about the perf impact.